openfisca_us/tests/baseline/finance/benefit/school_meals.yaml

```diff
@@ -10,13 +10,15 @@
     spm_units:
       spm_unit:
         members: [parent, child]
-        spm_unit_state_group: CONTIGUOUS_US
-        poverty_ratio: 0 
+        poverty_ratio: 0
+    households:
+      household:
+        state_group: CONTIGUOUS_US
   output:
     school_meal_subsidy: 180 * (3.66 + 1.97)
 
 
-- name: Low income, one parent, one child, contiguous_US
+- name: Low income, one parent, one child, contiguous_US (defined as default)
```

Comment:
This test breaks without defining `state_group` explicitly. @nikhilwoodruff should this work?

Reply:
Specifying the members of the household doesn't fix the error, but it does goes away if I specify `state_group` in the household, as per the other tests.